pkg/scuzz/ata_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package scuzz
+
+import (
+	"testing"
+)
+
+func TestAtaString(t *testing.T) {
+	const want = "Copyright 2019 the u"
+	var ts ataString
+	for i := 0; i < len(ts); i++ {
+		ts[i] = uint16(want[i*2+1])<<8 | uint16(want[i*2])
```

Comment:
the test should do this manually -- can you just write out by hand what you expect

Reply:
I can do it but it actually is harder to read. But, ok.